internal/pkg/template/template.go

```diff
@@ -264,6 +264,27 @@ func (t *Template) Test(testName string) (*Test, error) {
 	return &Test{name: testName, tmpl: t, fs: testDir}, nil
 }
 
+func (t *Template) CreateTest(testName string) (*Test, error) {
```

Comment:
Optional: Spravil by som novy typ `NewTest` (mozno ti napadne lepsi nazov).
```
type NewTest struct {
  *Test
}
```

A nad ten by som definoval nove metody: `SaveInputs`, `SaveExpectedOutput`.

Vysledkom by bolo, ze sa to neda pouzit nespravne, tj. nikto nezavola `SaveInputs` nad existujucim testom.
+ ak robis nieco nad testom, tak tam nebudes mat zbytocne metody v IDE `autocomplete`, ktore ani nemas pouzit.

Reply:
Aha, jo jo, to dává smysl. 👍